doc/install.rst

```diff
@@ -165,3 +152,25 @@ For more thorough testing, navigate back to the you can run the unit tests:
 
 After that, it's worth having a look at the :doc:`contribute page
 <contribute>`!
+
+Updating to the latest version
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When new commits are merged into the `master branch
+<https://github.com/ComPWA/tensorwaves/tree/master>`_, you need to update your
+:ref:`local copy of the source code <local-repository>` as follows:
+
+.. code-block:: shell
+
+  git checkout master
+  git pull
+  pip install -e .
```

Comment:
I believe the pip install -e is not even necessary if you previously installed it in editable mode

Reply:
Yeah but see what I [wrote underneath](https://github.com/ComPWA/tensorwaves/pull/116/files#diff-b50754a7065a1dece18f634c6b5b2633R171-R172): it's for potential changes in the dependencies

Reply:
I see